### ISSUE TITLE
Lesson 6: 26. Quiz submit issue

### ISSUE THREAD

**Issue by Spikhalskiy** [udacity/aind-issue-reports#20]
Have an issue with Advanced Game Playing # Quiz: Probabilistic Alpha-Beta Pruning. I'm trying to submit an answer which looks right. See screenshot: https://www.dropbox.com/s/192db4pxr81kx83/quiz.png And getting "Try again".

**Comment by danainschool** [comment]
This answer looks perfectly correct, and when I do the same I get no error.  If you close the browser and redo the quiz do you still get the error?  I am wondering if a space somewhere is causing this problem.  For example, if I type a "space" in one of the boxes that are supposed to be blank, I also get a "Try Again!"

**Comment by Spikhalskiy** [comment]
Hi, Dana.

I don't have a reproducible sequence. Only one I can tell - I tried to fill this quiz on iPad, understood that it's absolutely not adopted for mobile and switched to full Mac and got this resulting state.

On the same time:
> For example, if I type a "space" in one of the boxes

I think it's already a bug and should be submitted to your tech team. They could preprocess and cleanup field from spaces, non-ASCII symbols, whatever and it's easy. A server should be as much tolerant as possible to a client input.

**Comment by danainschool** [comment]
Agreed!  Thanks for your help - just trying to narrow the cause as much as possible so I know exactly who needs to fix it.  I am setting up an internal bug note. [J648]
